athena-oracle/src/main/java/com/amazonaws/athena/connectors/oracle/OracleMetadataHandler.java

```diff
@@ -207,6 +223,34 @@ public GetSplitsResponse doGetSplits(
         return new GetSplitsResponse(getSplitsRequest.getCatalogName(), splits, null);
     }
 
+    /**
+     * Overridden this method to describe the types of capabilities supported by a data source
+     * @param allocator Tool for creating and managing Apache Arrow Blocks.
+     * @param request Provides details about the catalog being used.
+     * @return A GetDataSourceCapabilitiesResponse object which returns a map of supported capabilities
+     */
+    @Override
+    public GetDataSourceCapabilitiesResponse doGetDataSourceCapabilities(BlockAllocator allocator, GetDataSourceCapabilitiesRequest request)
+    {
+        Set<StandardFunctions> unsupportedFunctions = ImmutableSet.of(NULLIF_FUNCTION_NAME, IS_DISTINCT_FROM_OPERATOR_FUNCTION_NAME, MODULUS_FUNCTION_NAME, ARRAY_CONSTRUCTOR_FUNCTION_NAME);
```

Comment:
same comments for array function and corresponding parser that i left on cloudera-hive